CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(SOURCE_FILES    src/Camera/Camera.cc
                     src/Dispatcher/Dispatcher.cc
                     src/Engine/Color.cc
                     src/Engine/Engine.cc
+                    src/Mesh/Mesh.cc
```

Comment:
No test file for mesh?

Reply:
Tested in Triangle as it is an abstract class